### PR TITLE
Simplify Copy of Newer Files / Script Cleanup

### PR DESCRIPTION
**Pull request by rkresha**
can just use cp -pu instead of the rsync command. Using cp makes it more
portable since it's part of the core tools, whereas rsync may require an additional install in some
distros.

### CODE REVIEW

**Comment by scottleibrand** [comment]
LGTM, just need someone to run the modified oref0-setup install and confirm everything still works on raspbian and ubilinux.

**Comment by rkresha** [comment]
Verified it works with raspbian

File: bin/oref0-setup.sh [hunk 1]
```diff
@@ -192,7 +192,7 @@ else
     (cd ~/src && git clone -b dev git://github.com/openaps/oref0.git) || die "Couldn't clone oref0 dev"
 fi
 echo Checking oref0 installation
-( grep -q oref0_glucose_since `which nightscout` && oref0-get-profile --exportDefaults 2>/dev/null >/dev/null ) || (echo Installing latest oref0 dev && cd $HOME/src/oref0/ && npm run global-install)
+( grep -q oref0_glucose_since $(which nightscout) && oref0-get-profile --exportDefaults 2>/dev/null >/dev/null ) || (echo Installing latest oref0 dev && cd $HOME/src/oref0/ && npm run global-install)
```
Comment (scottleibrand) [inline bin/oref0-setup.sh]: What's the benefit of that over ` syntax?

**Comment by scottleibrand** [comment]
I tested out the `cp -up` stuff with https://github.com/openaps/oref0/pull/235 and it worked fine for me, so I'm ready to merge that if you are.

File: bin/oref0-setup.sh (hunk 1 above)
Comment (rkresha) [inline bin/oref0-setup.sh]: per http://wiki.bash-hackers.org/scripting/obsolete

> \`COMMANDS\`	replaced by $(COMMANDS)	This is the older Bourne-compatible form of the command substitution. Both the \`COMMANDS\` and $(COMMANDS) syntaxes are specified by POSIX, but the latter is _greatly_ preferred, though the former is unfortunately still very prevalent in scripts. New-style command substitutions are widely implemented by every modern shell (and then some). The only reason for using backticks is for compatibility with a real Bourne shell (like Heirloom). Backtick command substitutions require special escaping when nested, and examples found in the wild are improperly quoted more often than not.

Comment (scottleibrand) [inline bin/oref0-setup.sh]: Ah, yes.  Quotes are one of the banes of my existence.  :-)

**Pull request merged**